shapepipe/modules/mccd_package/mccd_interpolation_script.py

```diff
@@ -389,7 +394,13 @@ def _interpolate_me(self):
         cat.open()
 
         all_id = np.copy(cat.get_data()['NUMBER'])
-        n_epoch = np.copy(cat.get_data()['N_EPOCH'])
+        key_ne = 'N_EPOCH'
+        if key_ne not in cat.get_data():
+            raise KeyError(
+                f'Key {key_ne} not found; run previous module (SExtractor)'
```

Comment:
Perhaps you can specify here where the code is looking for this key.

Reply:
Done